integration-test/src/test/scala/com/cloudera/livy/test/InteractiveIT.scala

```diff
@@ -69,9 +69,9 @@ class InteractiveIT extends BaseIntegrationTestSuite {
       s.run("sqlContext").verifyResult(startsWith("<pyspark.sql.context.HiveContext"))
       s.run("sc.parallelize(range(100)).map(lambda x: x * 2).reduce(lambda x, y: x + y)")
         .verifyResult("9900")
-      s.run("from pyspark.sql.types import Row").verifyResult(".*")
+      s.run("from pyspark.sql.types import Row").verifyResult("")
       s.run("x = [Row(age=1, name=u'a'), Row(age=2, name=u'b'), Row(age=3, name=u'c')]")
```

Comment:
Sorry mind if you explain what you are fixing?
I ran these code without your fix but %table magic is still working.